### PR TITLE
end2endtest: small fixes and add some comments

### PR DESCRIPTION
**Pull request by mariajdab**

### CODE REVIEW

File: dockerfiles/testsuite/start_test.sh [hunk 1]
```diff
@@ -116,11 +116,11 @@ e2etest_tokentxs() {
 	e2etest tokentxs
 }
 
-e2etest_overwritelection () {
+e2etest_overwritelection() {
   e2etest overwritelection
 }
 
-e2etest_cenususizelection () {
```
Comment (altergui) [inline dockerfiles/testsuite/start_test.sh:123]: cenusus -> census

**Comment by p4u** [comment]
please rebase this and the rest of your PR @mariajdab

**Comment by mariajdab** [comment]
> please rebase this and the rest of your PR @mariajdab

Done

**Comment by p4u** [comment]
@altergui feel free to approve and merge

**Review by altergui — APPROVED** [review]
LGTM

**Pull request merged**